src/java/org/apache/cassandra/db/memtable/ShardedMemtableConfigMXBean.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.db.memtable;
+
+public interface ShardedMemtableConfigMXBean
+{
+    /**
+     * Adjust the shard count for sharded memtables that do not specify it explicitly in the memtable options.
+     * Changes will apply on the next memtable flush.
+     */
+    public void setShardCount(String numShards);
```

Comment:
This setter/getter pair could be called `set/getDefaultShardCount`

Reply:
Changed